nautobot_floor_plan/__init__.py

```diff
@@ -56,10 +57,12 @@ def ready(self):
         """Callback after app is loaded."""
         super().ready()
         from .signals import (  # pylint: disable=import-outside-toplevel
+            load_middleware,
```

Comment:
I don't think we need a signal for this. It looks like `middleware` is just an option on `NautobotAppConfig`: https://docs.nautobot.com/projects/core/en/stable/development/apps/api/nautobot-app-config/#optional-nautobotappconfig-attributes